openbb_platform/obbject_extensions/charting/openbb_charting/__init__.py

```diff
@@ -44,19 +57,18 @@ def __init__(self, obbject):
         """Initialize Charting extension."""
         # pylint: disable=import-outside-toplevel
         from openbb_core.app.model.charts.charting_settings import ChartingSettings
-        from openbb_core.app.model.obbject import OBBject
 
         self._obbject: OBBject = obbject
         self._charting_settings = ChartingSettings(
             user_settings=self._obbject._user_settings,  # type: ignore
             system_settings=self._obbject._system_settings,  # type: ignore
         )
-        self._handle_backend()
 
     @classmethod
     def indicators(cls):
         """Returns a list of the available indicators."""
-        return ChartIndicators.get_available_indicators()
+        # return ChartIndicators.get_available_indicators()
+        return IndicatorsParams()
```

Comment:
I don't see how this is helpful, practically or visually. As detailed above, inheriting from BaseModel does not do the job that needs to be done.

```
{'sma': FieldInfo(annotation=SMAIndicatorsQueryParams, required=False, default=SMAIndicatorsQueryParams(length=50, offset=0), description='Simple Moving Average Indicators Query Params.'),
 'ema': FieldInfo(annotation=EMAIndicatorsQueryParams, required=False, default=EMAIndicatorsQueryParams(length=50, offset=0), description='Exponential Moving Average Indicators Query Params.')}
```

The output of `IndicatorsParams.model_fields` looks like this:

![Screenshot 2024-04-03 at 9 32 31 AM](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/a1dde91c-37b9-4a4e-a59a-898d0d20227b)

It's not readable. I want this because it clearly shows exactly what you need to know and nothing else:

![Screenshot 2024-04-03 at 9 34 19 AM](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/4bb03260-e89a-4206-9a5b-f613652255ca)

The class needs to be initialized for the `__repr__` to generate. Even if `__repr__` == `__doc__` it still needs to be initialized to extract the information required to replace `__doc__`.